celestia/peers.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
ideally we don't modify this file :slightly_smiling_face:

Reply:
It is not ideal with last minute changes, commit that touched the file was sent yesterday. I will close this PR and make a new one as otherwise it won't be clean if that file really doesn't have to be touched.